modules/util/portal-tools-rest-builder-test-test/src/testIntegration/java/com/liferay/portal/tools/rest/builder/test/resource/v1_0/test/ScopedTestEntityResourceTest.java

```diff
@@ -173,6 +218,18 @@ protected void assertValid(ScopedTestEntity scopedTestEntity)
 		return testDeleteSiteScopedTestEntityByExternalReferenceCode_addScopedTestEntity();
 	}
 
+	@Override
+	protected ScopedTestEntity testGraphQLScopedTestEntity_addScopedTestEntity()
```

Comment:
Same here, you are including in an "ignore" commit some code to make other tests work. I think this should be splitted or better explained